Revit_Core_Engine/Query/ViewSolid.cs

```diff
@@ -38,12 +38,12 @@ public static partial class Query
         [Description("Returns a solid that represents the 3-dimensional extents of a given view.")]
         [Input("view", "View to compute the solids.")]
         [Output("solid", "Solid that represents the 3-dimensional extents of the input view.")]
-        public static Solid ViewSolid(this View view)
+        public static Solid ViewSolid(this View view, bool createIfNoCropBox = false)
```

Comment:
Maybe `createIfUncropped`? Also a good description that 1e+6 in directions perpendicular to view directions will be created etc.

Reply:
Done 👍